docs/governance/charter.md

```diff
@@ -0,0 +1,42 @@
+<!-- 
+A project charter discusses _what it is and why it exists_, a governance discusses _how it operates_.
+
+Examples
+
+https://docs.clearlydefined.io/docs/community/charter
+https://openssf.org/about/charter/
+https://github.com/cncf/foundation/blob/main/charter.md
+https://github.com/mochajs/mocha/blob/main/PROJECT_CHARTER.md
+https://github.com/nodejs/TSC/blob/main/TSC-Charter.md
+Combined charter - governance https://github.com/camaraproject/Governance/blob/main/ProjectCharter.md
+blog about charters https://opensource.org/blog/what-is-open-governance-drafting-a-charter-for-an-open-source-project
+-->
+
+# Project Charter
+
+## 1. Mission
+<!-- Describe the project's purpose and the problem it addresses. Include a short mission statement. -->
+The EESSI project aims to build a common software stack that is:
+- Cross-platform (laptop, Cloud VM, HPC Cluster)
+- Ready-to-use (served over the internet, just mount-and-go)
+- Optimized for a wide range of hardware architectures (CPU, GPU, interconnects)
+- Easily extendable
+
+## 2. Scope
+<!-- Define what is within the scope of the project and what is explicitly out of scope. -->
+EESSI will focus on creating a repository of software installations (software.eessi.io). This requires:
```

Comment:
`repository` is common technical terminology that usually means "package repository" (like RPMs), but actually means something else for us (CernVM-FS repository), so let's avoid that term (or be explicit), something like:
```suggestion
EESSI focuses on creating a shared stack of software installations (`software.eessi.io`) that is distributed through a [CernVM-FS](https://cernvm.cern.ch/fs/) repository.

This requires:
```